packages/integrations/src/captureconsole.ts

```diff
@@ -70,9 +70,8 @@ export class CaptureConsole implements Integration {
           });
         }
 
-        // this fails for some browsers. :(
-        if (originalConsoleLevel) {
-          Function.prototype.apply.call(originalConsoleLevel, global.console, args);
+        if (originalConsoleLevel && 'originalConsoleLevel' in global.console) {
+          global.console[originalConsoleLevel](...args);
```

Comment:
Thanks for the contribution! Unfortunately, we can't accept it, because it both causes a build error and breaks the functionality of the code.

Remember that `originalConsoleLevel` is a function, which is why the build throws this error:

![image](https://user-images.githubusercontent.com/14812505/152593411-9e262b15-92c3-461c-b902-a3e438294833.png)

Further, `originalConsoleLevel` is not a property of `global.console`. I suspect you're thinking of `level` instead, which is the name of the method in question (`"warn"`, `"error"`, and so on), which IS a property of `global.console`. That said, changing `originalConsoleLevel` in your second line to `level` also doesn't fix it, because now `global.console[level]` is the wrapped function, not the original.

All of that said:

1) I can understand how you got confused, because `originalConsoleLevel` _sounds_ like it should be a string. I'll give it a better name.

2) You're not wrong that it can be simplified: it can be changed to `originalConsoleLevel.call(global.console, args);`, which I will also do.

Cheers!

Reply:
Thanks for the explanation, I didn't test or check the types yet (just made this in Github editor as an example). Thanks for taking it over and applying this change, this will save some errors in our sentry :)